ballista/core/src/config.rs

```diff
@@ -59,68 +58,51 @@ impl ConfigEntry {
         Self {
             name,
             _description,
-            _data_type,
+            data_type: _data_type,
             default_value,
         }
     }
 }
 
-/// Ballista configuration builder
-pub struct BallistaConfigBuilder {
-    settings: HashMap<String, String>,
```

Comment:
Oh, it looks like this was already a HashMap based thing. I guess this PR is refactoring that pattern?